src/api.ts

```diff
@@ -164,6 +165,21 @@ export class ScenarioApi {
     })
   })
 
+    // waits 30 seconds for consistency
+   simpleConsistency = async (instance_id, players: Array<Player>): Promise<Boolean> => {
+      var retries = 3
+      while (!await isConsistent(instance_id, players)) {
+        retries--
+        if (retries == 0) {
+          return false
+        }
+        await delay(10000)
+      }
+      return true
```

Comment:
Is there harm in doing more retries with a smaller interval, to spend less time waiting? Or does isConsistent take a long time itself?

Reply:
If getting state dumps is fairly quick, I'd be fine with replacing `s.consistency` with this, so there is only one way of doing it.

Reply:
The reason I didn't want to replace s.consistency is that going forward I didn't want people to believe that there was a generic way to determine consistency, that you actually have to write into your tests what that means to you if its anything beyond simple.